woocommerce/woocommerce
# [ci] Skip build for lint jobs

### Changes proposed in this Pull Request:

We don't need to build for linting. 
Set the build step to false in lint jobs. This should save a few good minutes with every lint job, depending on the project - around 4m for `plugin-woocommerce`.

### How to test the changes in this Pull Request:

Update some files that trigger a lint job. Make sure it runs successfully. For lint errors it fails (for the right reasons)

Tested with both JS and PHP errors: https://github.com/woocommerce/woocommerce/actions/runs/9403979444/job/25901884379
